tests/unit/challenges/test_models.py

```diff
@@ -115,25 +148,6 @@ class ChallengePhaseTestCase(BaseTestCase):
 
     def setUp(self):
         super(ChallengePhaseTestCase, self).setUp()
-        try:
```

Comment:
@TroJan, Since that Class ChallengePhase Class was to be used to be in 3 TestClass', I thought it would be good to move it into BaseClass because it would remove redundancy and reduce number of lines of code.